index.js

```diff
@@ -151,14 +151,16 @@
 
   /* istanbul ignore else */
   if (typeof module === 'object' && typeof module.exports === 'object') {
-    module.exports = f(require('ramda'), require('sanctuary-def'));
+    module.exports = f(require('ramda'),
+                       require('sanctuary-type-classes'),
+                       require('sanctuary-def'));
   } else if (typeof define === 'function' && define.amd != null) {
-    define(['ramda', 'sanctuary-def'], f);
+    define(['ramda', 'sanctuary-type-classes', 'sanctuary-def'], f);
   } else {
-    self.sanctuary = f(self.R, self.sanctuaryDef);
+    self.sanctuary = f(self.R, self.sanctuaryTypeClasses, self.sanctuaryDef);
   }
 
-}(function(R, $) {
+}(function(R, λ, $) {
```

Comment:
I'm not sure. I placed it in the `l` register in Vim so I was able to type <kbd>"</kbd> <kbd>l</kbd> <kbd>p</kbd> to insert it.

We should probably switch to an identifier that's easier to type. `_`, perhaps? I like one-character identifiers for dependencies we reference dozens of times. :)